index.html

```diff
@@ -38,97 +38,128 @@
-    <ul class="languages-1">
-      <li class="html">HTML/CSS</li>
-      <li class="ruby">Ruby on Rails</li>
-      <li class="js">JavaScript</li>
-    </ul>
-    <button type="button" class="projects_button">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    <div class="projectinfo4">
+      <h3 class="projectname">Project name goes here</h3>
+      <ul class="languages">
+        <li class="html">HTML/CSS</li>
+        <li class="ruby">Ruby on Rails</li>
+        <li class="js">JavaScript</li>
+      </ul>
+      <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    </div>
   </div>
 
-  <div class="description">
+  <div class="description fifth">
     <img src="images/proj5.png" alt="PROJECT5">
-    <h3 class="name">Project name goes here</h3>
-    <ul class="languages-1">
-      <li class="html">HTML/CSS</li>
-      <li class="ruby">Ruby on Rails</li>
-      <li class="js">JavaScript</li>
-    </ul>
-    <button type="button" class="projects_button">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    <div class="projectinfo5">
+      <h3 class="projectname">Project name goes here</h3>
+      <ul class="languages">
+        <li class="html">HTML/CSS</li>
+        <li class="ruby">Ruby on Rails</li>
+        <li class="js">JavaScript</li>
+      </ul>
+      <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    </div>
   </div>
-
-  <div class="description">
+  <div class="description sixth">
     <img src="images/proj6.png" alt="PROJECT6">
-    <h3 class="name">Project name goes here</h3>
-    <ul class="languages-1">
-      <li class="html">HTML/CSS</li>
-      <li class="ruby">Ruby on Rails</li>
-      <li class="js">JavaScript</li>
-    </ul>
-    <button type="button" class="projects_button">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    <div class="projectinfo6">
+      <h3 class="projectname">Project name goes here</h3>
+      <ul class="languages">
+        <li class="html">HTML/CSS</li>
+        <li class="ruby">Ruby on Rails</li>
+        <li class="js">JavaScript</li>
+      </ul>
+      <button type="button" class="buttonproject">See this Project<img src="images/arrow.png" alt="ARROW"></button>
+    </div>
   </div>
+  <div class="background1"></div>
+  <div class="background2"></div>
+  <div class="background3"></div>
 </section>
 
 <!-- About US section -->
-  <section class="intro">
+  <section class="intro about-intro">
+    <section class="about-description">
     <h5>About</h5>
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
     <button type="button" class="about-button">Get my CV<img src="images/dwnld.png" alt="Download"></button>  
   </section>
```

Comment:
- [ ] Please decrease the width of the `get my CV` button and sign it to the left as provided in Figma. 
- [ ] When I hover over the button the icon is not visible:
<img width="438" alt="Screenshot 2022-04-01 at 12 00 26" src="https://user-images.githubusercontent.com/77038610/161221176-c1e4f665-e6a6-48c1-ba8f-0910851cc4ad.png">
 

- [ ] I recommend deleting borders from all buttons to make the project look cleaner and adding a dark border only on hover.